### PR TITLE
Fix blast resistance values for all blocks

### PR DESCRIPTION
**Pull request by 501st-alpha1**
Fixes VANILLA-410
Some blocks didn't have any resistance set, most just had wrong value.

### CODE REVIEW

File: src/main/java/org/spout/vanilla/material/block/liquid/Lava.java [hunk 1]
```diff
@@ -47,6 +47,7 @@
 	public Lava(String name, int id, boolean flowing) {
 		super(name, id, flowing, VanillaMaterialModels.LAVA);
 		this.setFlowDelay(1500);
+		this.setResistance(flowing ? 0.0F : 500.0F);
```
Comment (olliestanley) [inline src/main/java/org/spout/vanilla/material/block/liquid/Lava.java:50]: Idk if it matters too much, but this should probably be this.setFlowDelay(1500).setResistance(flowing ? 0.0F : 500F);
(Same for Water.java)

Comment (kitskub) [inline src/main/java/org/spout/vanilla/material/block/liquid/Lava.java:50]: tbh, there are many of Vanilla's materials that are this way. But to be sure, @Zidane?

Comment (olliestanley) [inline src/main/java/org/spout/vanilla/material/block/liquid/Lava.java:50]: Meh, best to keep it from getting any worse I guess :P

**Comment by dredhorse** [comment]
I like the two liner

also tests are not working for this

**Comment by olliestanley** [comment]
It's designed for the setters to return the object so you can do that in the first place. It should at least be kept consistent one way or the other. But really, this can always be changed later.

**Comment by windy1** [comment]
Looks good, all we need to pull this is a sign-off to acknowledge you agree to the Certificate of Origin found [here](https://wiki.spout.org/Pull_request_guidelines#Certificate_of_origin). That is also the location of the broken link you posted in your post on spout.org as well.

You can edit a commit description by using `git commit --amend` and then adding `Signed-off-by: <you@example.com>` to the bottom of your description. You can update the commit here with `git push --force origin blast_resistance`.

**Comment by 501st-alpha1** [comment]
Okay, thanks. I'll take care of that when I get off work this evening. I can't figure out why the tests are failing, as they passed on my system. Ideas?

**Comment by windy1** [comment]
I'm not sure, but I think it's unrelated to this PR anyways.

File: src/main/java/org/spout/vanilla/material/block/liquid/Lava.java (hunk 1 above)
Comment (501st-alpha1) [inline src/main/java/org/spout/vanilla/material/block/liquid/Lava.java:50]: Yeah, I saw it both ways, so I wasn't sure which was preferred.

**Comment by kitskub** [comment]
Oops sorry. I meant to say that test failures ARE unrelated to this PR.

**Pull request merged**